### ISSUE TITLE
Last station's card in a hex's long station list is truncated in web explorer

### ISSUE THREAD

**Issue by Greg15** [WeatherXM/issue-tracker#124]
## WeatherXM Related Element
- Website (explorer.weatherxm.com)

## **Describe the bug**
When the list of displayed stations in a hex is long, part of the last station card is truncated

### **How To Reproduce**
1. Go to explorer.weatherxm.com
2. Drill down to a hex with lots of stations. The 2 likely candidates are the (0,0) hex and the hex that includes Athens.
3. Click on the hex to get the list of stations in the hex.
4. Scroll down to the bottom of the long list.
5. Notice that the last station's card is truncated (in the screenshot, the station is Witty Rosewood Solar)

### **Expected behavior**
The last station's card should fully display, even when the list is long.

### **Screenshots**
![Screenshot_20220815-032148.png](https://user-images.githubusercontent.com/70354350/184602294-dc18a399-c2f4-464d-a66c-d592e9f6584b.png)

### **Smartphone:**
 - Device: Moto g7 power 
 - OS: android version 10 
 - Browser: Chrome 103.0.5060.71
 - Android App Version: N/A

### **Additional context**
1. I accessed the explorer using chrome on my phone. I tried it both in standard mobile mode, as well as telling Chrome to use "Desktop site". The issue existed both ways.
2. A bug was previously submitted for the same behavior in the in-app explorer (2+ months ago?), which was already fixed quite a while ago. As a result, the logic to properly resolve this issue in the website explorer might be able to be borrowed from the android in-app explorer fix.

**Comment by PavlosTze** [comment]
@Greg15 This should be fixed in the latest explorer version we just released. Can you please confirm?

**Comment by Greg15** [comment]
Well, it's different, but not solved.
As I did originally, I tested the explorer using the Chrome browser on my Android phone.
Since I submitted the original issue, my phone's Chrome browser version has updated to: 105.0.5195.79

Before, I was seeing the same issue in both normal browser mode for a phone, and also when checking "Desktop site" in the phone browser settings.

Now, in normal mode, I think the bottom station doesn't appear at all, and the 2nd to bottom station is truncated.
And in "Desktop site", I can see them all, but the explorer map unfortunately only uses a little bit of the screen.

Normal phone browser mode:
![Screenshot_20220912-213439.png](https://user-images.githubusercontent.com/70354350/189795874-aaec8870-4855-412e-9000-fb7d64a62fcc.png)

"Desktop site" mode on phone browser:
![Screenshot_20220912-213608.png](https://user-images.githubusercontent.com/70354350/189795978-1a45c824-187f-4dea-b492-3a46aafb36a3.png)

**Comment by PavlosTze** [comment]
@Greg15 Can you please check again?

**Comment by Greg15** [comment]
I tried, but I'm not sure my check is adequate.
The 2 hexes that definitely had enough stations to cause the issue were the (0,0) hex and the Athens hex. The (0,0) hex no longer has any stations, and the Athens hex now only has 6 stations.

The Athens hex DOES correctly display with the current 6 stations.

I have no idea how many stations were needed to encounter the issue before, because I think any hex I encountered before had either no more than 3, or had at least 15.
